ci-operator/config/openshift/openshift-tests-private/.config.prowgen

```diff
@@ -27,6 +27,10 @@ slack_reporter:
   - gcp-ipi-sdn-fips-f7-logging
   - gcp-ipi-sdn-fips-f14-logging
   - gcp-ipi-sdn-fips-f28-logging
+  - aws-ipi-arm-f7-logging
+  - azure-ipi-oidc-f30-logging
+  - gcp-ipi-fips-f30-logging
+  - azure-ipi-arm-f30-loggin
```

Comment:
```suggestion
  - azure-ipi-arm-f30-logging
```

Reply:
BTW, seems there is no job named `azure-ipi-arm-f30-logging`, should it be removed?